tests/ui/parser/issues/unnessary-error-issue-138401.stderr

```diff
@@ -0,0 +1,10 @@
+error: mismatched closing delimiter: `)`
+  --> $DIR/unnessary-error-issue-138401.rs:1:27
+   |
+LL | pub fn foo(x: i64) -> i64 {
+   |                           ^ unclosed delimiter
+LL |     x.abs)
+   |          ^ mismatched closing delimiter, may missing open `(`
```

Comment:
it's better to suggest like this:
```
missing open `(` for this delimiter
```

as the origin issue reported wrote in https://github.com/rust-lang/rust/issues/138401